bootstrapvz/common/tasks/grub.py

```diff
@@ -62,9 +78,9 @@ def run(cls, info):
 				if not isinstance(p_map, partitionmaps.none.NoPartitions):
 					for idx, partition in enumerate(info.volume.partition_map.partitions):
 						device_map.write('(hd0,{prefix}{idx}) {device_path}\n'
-						                 .format(device_path=partition.device_path,
-						                         prefix=partition_prefix,
-						                         idx=idx + 1))
+										 .format(device_path=partition.device_path,
+												 prefix=partition_prefix,
+												 idx=idx + 1))
```

Comment:
Again, please align with spaces instead of tabs (which are used for indentation only).